addons/surgery/XEH_preInit.sqf

```diff
@@ -157,7 +139,7 @@ PREP_RECOMPILE_END;
     "LIST",
     LLSTRING(CLOSED_REDUCTION_MEDLEVEL),
     [CBA_SETTINGS_CAT, LSTRING(SubCategory_SurgicalActions)],
-    [[0, 1, 2], ["STR_ACE_Medical_Treatment_Anyone", "STR_ACE_Medical_Treatment_Medics", "STR_ACE_Medical_Treatment_Doctors"], 0],
+    [[0, 1, 2], ["STR_ACE_Medical_Treatment_Anyone", "STR_ACE_Medical_Treatment_Medics", "STR_ACE_Medical_Treatment_Doctors"], 2],
```

Comment:
i wouldn't touch it personally, options are made to be changed (and iirc we kinda try to keep defaults as anyone to lessen "bug reports")